### PR TITLE
Update app-templates.md

### PR DESCRIPTION
**Pull request by jhwang-ms**
Adding two additional adoption tools to this page, submitting on behalf of Josh Leporati.

### CODE REVIEW

**Comment by jhwang-ms** [comment]
@Mahesh2516 and @KirtiPereira - kindly request a status update on this review request so that I can provide a status update for my manager. Thank you.

File: msteams-platform/samples/app-templates.md [hunk 1]
```diff
@@ -23,6 +23,18 @@ All app templates are provided under the [MIT License](https://github.com/Office
 * **Customizable and extensible:** While all app templates can be deployed with minimal configuration, we provide the entire code base and deployment scripts so that you can easily customize or extend them to fit your unique needs.
 * **Detailed documentation:** All app templates are accompanied by end-to-end documentation on solution architecture, deployment, and configuration steps.  
 
+## Adoption tool - Champion Management Platform &#9734;
+
+The Champion Management Platform (CMP) app template helps you manage, scale, and inspire your teamwork champions to achieve more. This app template is built on the SharePoint Framework and loaded into a tab within a team. Groups can leverage this tool to help manage program membership, provide a leaderboard and event types for logging, as well as tooling to overlay digital badges to program participants.
```
Comment (Mahesh2516) [inline msteams-platform/samples/app-templates.md:28]: The sentence can be written as:

Groups can leverage this tool and manage program membership, provide a leaderboard and event types for logging, and tools to overlay digital badges to program participants.

**Review by Mahesh2516 — CHANGES_REQUESTED** [review]

**Review by KirtiPereira — APPROVED** [review]
@jhwang-ms - we have approved the PR and it will be pushed to live.
Approved

**Review by surbhigupta12 — DISMISSED** [review]
Approved the PR

**Review by Mahesh2516 — DISMISSED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Review by KirtiPereira — APPROVED** [review]
Approved

**Review by Mahesh2516 — APPROVED** [review]
Approved

**Comment by jhwang-ms** [comment]
@KirtiPereira many thanks for pushing this live, but I see that the wording layout was changed. My manager, Karuana Gatimu, owns these two adoption tools and wants them to read the way we had submitted them originally: "Adoption tool - Change Management Platform" and "Adoption tool - Microsoft 365 learning pathways app (Get Started)". Can we revert please?

**Pull request closed without merging**